hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSourceWALReaderThread.java

```diff
@@ -511,9 +587,9 @@ public boolean isEmpty() {
       return walEntriesWithSize.isEmpty();
     }
 
-    public void updatePosition(WALEntryStream entryStream) {
-      lastWalPath = entryStream.getCurrentPath();
-      lastWalPosition = entryStream.getPosition();
+    public void updatePosition(Path currentPath, long currentPosition) {
```

Comment:
can you add javadoc on this method to make it clear what's expected for these parameters since it's a public method?